src/defs.asm

```diff
@@ -16,6 +16,9 @@ ENDIF
 
 ; ---------------------------------------------------------------------------
 
+AreaIndex_Start = $00
```

Comment:
```suggestion
; Each level has a maximum of 10 ($00~$09) areas.
; Extra ones are generally empty or pointers to the last 'real' area.
; "Common jars", if used in a level, occupy area $04.
; but if common jars aren't used, it can (and often is) a regular area.
AreaIndex_Start = $00
```

This comment could probably be repeated and expanded upon elsewhere (perhaps in the level data documentation), too.